README.rst

```diff
@@ -83,6 +83,16 @@ to one of the following:
 
 To learn more about the loader, see `here <https://github.com/yaml/pyyaml/wiki/PyYAML-yaml.load(input)-Deprecation>`_
 
+TOML
+~~~~~~~~~~
+
+`TOML`_ TOML aims to be a minimal configuration file format that's easy to read due to obvious semantics. TOML is designed to map unambiguously to a hash table.
```

Comment:
```suggestion
`TOML`_ aims to be a minimal configuration file format that's easy to read due to obvious semantics. TOML is designed to map unambiguously to a hash table.
```

Also, please add a proper link at the bottom of the file (compare with `YAML`). 👍